data/supporters.json

```diff
@@ -515,7 +515,21 @@
           "twitter": "JohannesHoppe"
         }
       ]
-    }
+    },
+    {
+      "name": "Angular Berlin",
+      "city": "Berlin",
+      "country": "Germany",
+      "link": "https://www.meetup.com/de-DE/AngularJS-Meetup-Berlin/",
+      "twitter": "angular_berlin",
+      "contacts": [
+        {
+          "name": "Martin Sonnenholzer",
+          "email": "martin@angular.berlin",
+          "twitter": "chaos_monster"
+        }
+      ]
+    },
```

Comment:
This is not valid JSON (trailing comma not allowed).

Could you please check again locally and change? Thank you!